src/sentry/static/sentry/app/views/settings/components/forms/rangeField.jsx

```diff
@@ -0,0 +1,148 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import styled from 'react-emotion';
+
+const Slider = styled.input`
+  -webkit-appearance: none;
+  width: 100%;
+  margin: ${p => p.theme.grid}px 0;
+
+  &:focus {
+    outline: none;
+    &::-webkit-slider-runnable-track {
+      background: ${p => p.theme.borderDark};
+    }
+  }
+
+  &::-webkit-slider-runnable-track {
```

Comment:
high level comment on this stuff: I'm seeing a lot of duplication between the different browser-specific pseudo elements. It might be worth it to group some of these together, doing something like:

```css
&::-webkit-slider-runnable-track, &::-moz-range-track, input[type=range]::-ms-track {
  width: 100%;
  height: 3px;
  cursor: pointer;
}
```

would save 6 lines of css and would also help me to understand what css is shared and what isn't. 

Similarly it seems like we can save a ton of code by combining `&::-webkit-slider-thumb`, `&::-moz-range-thumb`, & `input[type=range]::-ms-thumb` which have almost identical css so that would save probably 18 lines of code or so!

Reply:
Yeah good call. The `input[type=range]` stuff should be there in the first place. I pasted them and forgot to translate to &:: about halfway through it appears.